lib/libutee/tee_api_operations.c

```diff
@@ -763,39 +763,17 @@ static TEE_Result tee_buffer_update(
 
 	if (slen >= (buffer_size + buffer_left)) {
 		/* Buffer is empty, feed as much as possible from src */
-		if (TEE_ALIGNMENT_IS_OK(src, uint32_t)) {
```

Comment:
@pascal-brand-st-dev regarding accelerated ciphers: the only accelerated cipher we have currently is AES on arm64. The functions will accept unaligned data buffers, because they load the SIMD registers using byte elements like `ld1 {v0.16b}, [x0]`. Same goes for data storage with the `st1` instruction. So, it would seem we do not need to enforce any specific alignment on the data buffers.